tensorflow/core/common_runtime/bfc_allocator.cc

```diff
@@ -295,16 +294,25 @@ bool BFCAllocator::DeallocateFreeRegions(size_t rounded_bytes) {
     return false;
   }
 
-  VLOG(INFO) << "Re-allocate memory regions to avoid OOM due to memory"
-             << " fragmentation. If you see this message frequently, note"
-             << " that the re-allocation may incur performance overhead despite"
-             << " better memory utilization. You may try smaller batch sizes"
-             << " to see if it can give you better performance.";
+  VLOG(WARNING) << "Re-allocate memory regions (i.e., allocations) to avoid OOM"
```

Comment:
LOG(WARNING) ?

Reply:
I also just noticed that myself but good catch!